src/client.ts

```diff
@@ -903,6 +943,19 @@ export class ConnectionImpl {
 
         case chars.$Z: {
           this._parseSyncMessage();
+
+          if (
+            !(this.sock instanceof tls.TLSSocket) &&
+            // @ts-ignore
+            typeof Deno === "undefined" &&
```

Comment:
I'm not sure this is the best way to special case deno :thinking:

Reply:
This looks reasonable to me - any comments @jaclarke or @1st1 pls?

Reply:
I think it's ok, we already use that check in other places: https://github.com/edgedb/edgedb-js/blob/master/src/compat.ts#L44